notes-app-client/src/NotesApp.jsx

```diff
@@ -38,9 +38,26 @@ function NotesApp() {
     setActiveNote(newNote.id);
   };
 
-  const onDeleteNote = (noteId) => {
+  const onDeleteNote = async (noteId) => {
+
+    // const deleteResponse = await fetch(
+    //   '/api/notes/delete',
+    //   {
+    //     method: "DELETE",
+    //   }
+    // );
+
+        try {
+            let response = await fetch(`/api/notes/${noteId}`, {
+                method: "DELETE",
+            }); 
+        } catch (err) {
+    }
+
```

Comment:
Once the note is deleted you can re-fetch the notes from the backend instead of doing a local filter like in line :57